### DISCUSSION TITLE
Is is smart to use the domain field as tenant ID

### DISCUSSION THREAD

**Discussion by HenkVanMaanen** [plausible/analytics#2903] [Anything Else]
We are a SaaS business and would like to keep track of customer statistics, but the data should be retrievable per customer ID, aka tenant ID. What are the drawbacks of creating a new website in plausible per customer / tenant ID, instead of creating one website in plausible and then seperating customers by custom props in events.

**Comment by metmarkosaric** [comment]
thanks for reaching out @HenkVanMaanen! from our side it's all flexible so the approach to take is more up to your needs. the domain name field doesn't necessary need to correspond to an actual domain name so you can call it whatever you want if you wish to do so. one of the benefits of having individual dashboards for individual customers is that it makes it easier to share the stats with them if you wish to do so using our embed dashboard functionality. there's a bit [more info on this here](https://plausible.io/white-label-web-analytics). we have a free trial that allows you to test different approaches and see what makes the most sense for you.

**Reply by RobertJoonas** [reply]
@HenkVanMaanen, having separate dashboards is totally fine by us, but if you'd like to keep everything in one dashboard, that's possible too. We've just released a private preview of the custom pageview properties feature. Please reach out to hello@plausible.io with all email addresses of people in your team, and I can enable the feature for your accounts :)

**Reply by RobertJoonas** [reply]
Here's the documentation for how those properties can be sent to Plausible: https://plausible.io/docs/custom-pageview-props

**Reply by HenkVanMaanen** [reply]
Sounds good, thanks for the help!
